src/components/ProductTours/data/hooks.js

```diff
@@ -0,0 +1,40 @@
+import { useContext, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform/config';
+import Cookies from 'universal-cookie';
+import { ROUTE_NAMES } from '../../EnterpriseApp/constants';
+import {
+  BROWSE_AND_REQUEST_TOUR_COOKIE_NAME,
+  LEARNER_CREDIT_COOKIE_NAME,
+} from '../constants';
+import { features } from '../../../config';
+import { SubsidyRequestsContext } from '../../subsidy-requests';
+import { EnterpriseSubsidiesContext } from '../../EnterpriseSubsidiesContext';
+
+const cookies = new Cookies();
+
+export const useBrowseAndRequestTour = () => {
+  const history = useHistory();
+  const inSettingsPage = history.location.pathname.includes(ROUTE_NAMES.settings);
+  const { subsidyRequestConfiguration } = useContext(SubsidyRequestsContext);
+  const isBrowseAndRequestEnabledForEnterprise = features.FEATURE_BROWSE_AND_REQUEST;
+  const dismissedBrowseAndRequestTourCookie = cookies.get(BROWSE_AND_REQUEST_TOUR_COOKIE_NAME);
+  // Only show tour if feature is enabled, the enterprise is eligible for the feature,
+  // hide cookie is undefined or false, not in settings page, and subsidy requests are not already enabled
+  const showBrowseAndRequestTour = isBrowseAndRequestEnabledForEnterprise
+    && !dismissedBrowseAndRequestTourCookie && !inSettingsPage && !subsidyRequestConfiguration?.subsidyRequestsEnabled;
+  return useState(showBrowseAndRequestTour);
```

Comment:
nit: I think it could be helpful to:
1. do the array destructing of `useState` inside the hook and return an explicit array, so it's more obvious that this will return an array with the state value and state setter function at first glance.
2. and/or add a JSDoc to these hook functions to describe the return value a bit more explicitly.